scripts/launch-local-binary.sh

```diff
@@ -99,6 +99,7 @@ sleep 10
 $PARACHAIN_BIN --alice --collator --force-authoring --tmp --chain $CHAIN-dev \
   --unsafe-ws-external --unsafe-rpc-external --rpc-cors=all \
   --port 30333 --ws-port 9944 --rpc-port 9933 --execution wasm \
+  --state-pruning archive --blocks-pruning archive \
```

Comment:
Can you please also add this to the config yml file: `docker/*-parachain-launch-config.yml`?

Reply:
sure.